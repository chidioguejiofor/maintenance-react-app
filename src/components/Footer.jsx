import React from 'react';
import SocialLogoGrid from './SocialLogoGrid';

const Footer = () => (
  <footer className="simple-background-image">
    <h1>Credits</h1>
    <p>
      This app was created by Chidiebere Oguejiofor
        during his  training period at Andela

    </p>
    <p>
    Special thanks to my Learning Facilitators and those who
    have helped me in my journey to this point
    </p>

    <SocialLogoGrid />
  </footer>
);

export default Footer;
