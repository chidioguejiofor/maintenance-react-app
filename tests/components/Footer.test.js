import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../src/components/Footer';

describe('Testing Footer', () => {
  describe('Testing FooterContainer', () => {
    it('should match the existing snapshot of the app ', () => {
      const wrapper = shallow(<Footer />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
