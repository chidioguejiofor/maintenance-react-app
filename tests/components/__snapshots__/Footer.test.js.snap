// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer Testing FooterContainer should match the existing snapshot of the app  1`] = `
<footer
  className="simple-background-image"
>
  <h1>
    Credits
  </h1>
  <p>
    This app was created by Chidiebere Oguejiofor during his  training period at Andela
  </p>
  <p>
    Special thanks to my Learning Facilitars and those who have helped me in my journey to this point
  </p>
  <SocialLogoGrid />
</footer>
`;
